txclib/commands.py

```diff
@@ -107,7 +114,7 @@ def cmd_set(argv, path_to_tx):
             wizard_options = Wizard(path_to_tx).run()
             wizard_options.update(default_options)
             options = Namespace(**wizard_options)
-            parser = set_parser()
+            parser = set_parser(is_set=is_set)
```

Comment:
@SofiaMargariti it took me a while to understand what `is_set` is. I'd personally prefer the argument to be named something like `is_legacy`, but I won't insist.

Reply:
Ok I will update it, I too, think it is better.